### PR TITLE
All tests in handler are passing

### PR DESCRIPTION
**Pull request by StefanGajic**
https://trello.com/c/lmBQdE1S/137-fix-tests-in-handlers-which-are-not-passing

### CODE REVIEW

File: handler/bookhandler_test.go [hunk 1]
```diff
@@ -105,7 +105,7 @@ func TestIndex(t *testing.T) {
 		handler := bookHandler.Index
 		httperror := handler(rr, req)
 
-		expectedResponse := "HTTP 500: Error finding books"
+		expectedResponse := "HTTP 500: Error finding books: "
```
Comment (sfqi) [inline handler/bookhandler_test.go]: why is this happening now?
why do all errors end with `: `?

Comment (StefanGajic) [reply handler/bookhandler_test.go]: I think it is because we added `:` 
I tried to remove colon but actual needs `:`  at the end

Comment (bokimilinkovic) [reply handler/bookhandler_test.go]: this should be optional, if we have some additional context for error message

Comment (sfqi) [reply handler/bookhandler_test.go]: If every error message ends with `: `, we should fix that. As in, they shouldn't all end with `: `

Comment (sfqi) [reply handler/bookhandler_test.go]: Let's close this PR and deal with the issue properly

**Pull request closed without merging**